### ISSUE TITLE
License selection and inclusion in ttkthemes package

### ISSUE THREAD

**Issue by RedFantom** [MaxPerl/ttk-Breeze#2]
Hey MaxPerl!

Your ttk theme looks very nice. If you don't mind, I would like to include it in [`ttkthemes`](https://github.com/RedFantom/ttkthemes), a Python package which contains as many ttk themes as I can find. Currently, you don't have a license selected for your project, so I can't legally include it in the package. Do you object if I include it? Would you mind choosing a license for your theme? Licenses that would allow inclusion include:
- GNU GPLv2/3
- MIT License
- Apache 2.0 license
- The Tcl-license

Please let me know what you think. Your theme would be a great addition to the package for all the KDE-users out there.

**Comment by MaxPerl** [comment]
Dear RedFantom,
it would be a honour ;-) I created LGPL 2.1 license file. I hope this is okay...
Best wishes,
Max

**Comment by RedFantom** [comment]
Thank you! The LGPL v2.1 is compatible with the GNU GPLv3 (in only a single direction, but the one I need), so now I can work on including this theme, which is great. I have already pushed the initial version of the theme to a [branch](https://github.com/RedFantom/ttkthemes/tree/breeze), but I have noticed some small issues, so I will look into them there. If I make any changes that are worth merging back, I will make sure to open a PR here.
I'll let you know once I get everything set up to be included in the next release!
